### DISCUSSION TITLE
Support CIS kubernetes CIS-1.10 for k8s v1.28 - v1.31

### DISCUSSION THREAD

**Discussion by andypitcher** [aquasecurity/kube-bench#1692] [Development]
[CIS Kubernetes Benchmark v1.10.0](https://workbench.cisecurity.org/benchmarks/17568) (targets k8s v1.28 - v1.31) has been released in CIS Workbench.

@mozillazg @afdesk happy to take care of it, if you could create a placeholder issue **Support CIS kubernetes CIS-1.10 for k8s v1.28 - v1.31** as this one: [Support CIS kubernetes CIS-1.9 for k8s v1.27 - v1.29](https://github.com/aquasecurity/kube-bench/issues/1616)

**Comment by afdesk** [comment]
Hi guys! 
@andypitcher @mozillazg You're so cool! thanks a lot!

I've created #1693 to track it
